soca/soca_setcorscales.yaml

```diff
@@ -7,31 +7,31 @@ date: 2019-08-31T12:00:00Z
 corr variables: [socn, tocn, ssh, hocn, cicen, hicen, hsnon]
```

Comment:
Don't commit that file just yet. We'll update when we have a better setup, which isn't the case right now.